crates/tracing/src/lib.rs

```diff
@@ -223,8 +226,5 @@ impl Tracer for RethTracer {
 ///
 ///  The subscriber will silently fail if it could not be installed.
 pub fn init_test_tracing() {
-    let _ = tracing_subscriber::fmt()
-        .with_env_filter(EnvFilter::from_default_env())
-        .with_writer(std::io::stderr)
-        .try_init();
+    TestTracer.init().unwrap();
```

Comment:
this should not panic (equivalent to previous behaviour)
```suggestion
   let _ = TestTracer::default().init();
```